plugin/v2/settable_test.go

```diff
@@ -89,3 +92,47 @@ func TestUpdateConfigEnv(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateConfigMounts(t *testing.T) {
+	source := "source"
+	foo := "foo"
+
+	contexts := []struct {
+		mounts    []types.PluginMount
+		set       settable
+		newMounts []types.PluginMount
+	}{
+		{[]types.PluginMount{types.PluginMount{Name: "name", Source: &source}}, settable{name: "name", value: "foo"}, []types.PluginMount{types.PluginMount{Name: "name", Source: &foo}}},
```

Comment:
the cool thing is you don't even need `types.PluginMount` you can do: `[]types.PluginMount{{Name: "name", Source: &source}}`